### PR TITLE
:zap: Download images and files.

### PR DESCRIPTION
**Pull request by RicardoE105**
@janober requested by @tanay1337

### CODE REVIEW

File: packages/nodes-base/nodes/Telegram/TelegramTrigger.node.ts [hunk 1]
```diff
@@ -105,6 +110,48 @@ export class TelegramTrigger implements INodeType {
...
+				type: 'collection',
+				placeholder: 'Add Field',
+				default: {},
+				options: [
+					{
+						displayName: 'Download Images/Files',
+						name: 'donwload',
+						type: 'boolean',
+						default: false,
+						description: `Telegram develiers the image in 3 sizes.<br>
```
Comment (janober) [inline packages/nodes-base/nodes/Telegram/TelegramTrigger.node.ts:125]: Looks like a typo

File: packages/nodes-base/nodes/Telegram/TelegramTrigger.node.ts [hunk 2]
```diff
@@ -105,6 +110,48 @@ export class TelegramTrigger implements INodeType {
...
+			{
+				displayName: 'Additional Fields',
+				name: 'additionalFields',
+				type: 'collection',
+				placeholder: 'Add Field',
+				default: {},
+				options: [
+					{
+						displayName: 'Download Images/Files',
+						name: 'donwload',
```
Comment (janober) [inline packages/nodes-base/nodes/Telegram/TelegramTrigger.node.ts:122]: also typo

File: packages/nodes-base/nodes/Telegram/TelegramTrigger.node.ts [hunk 3]
```diff
@@ -105,6 +110,48 @@ export class TelegramTrigger implements INodeType {
...
+						type: 'boolean',
+						default: false,
+						description: `Telegram develiers the image in 3 sizes.<br>
+						By default, just the larger image would be downloaded.<br>
+						if you want to change the size set the field 'Image Size'`,
+					},
+					{
+						displayName: 'Image Size',
+						name: 'imageSize',
```
Comment (janober) [inline packages/nodes-base/nodes/Telegram/TelegramTrigger.node.ts:131]: Should it not only be displayed if "download" is true?

Comment (RicardoE105) [reply packages/nodes-base/nodes/Telegram/TelegramTrigger.node.ts:131]: Yes, will add the fix tomorrow. Thanks

**Pull request merged**